tests/tests/gas_price.rs

```diff
@@ -203,23 +276,81 @@ async fn produce_block__lowers_gas_price() {
     let _ = client.produce_blocks(1, None).await.unwrap();
 
     // then
-    let change = starting_gas_price * percent / 100;
+    let change = starting_gas_price * percent as u64 / 100;
     let expected = starting_gas_price - change;
     let latest = client.latest_gas_price().await.unwrap();
     let actual = latest.gas_price;
     assert_eq!(expected, actual);
 }
 
+#[tokio::test]
+async fn produce_block__raises_gas_price_with_default_parameters() {
+    // given
+    let args = vec![
+        "--debug",
+        "--poa-instant",
+        "false",
+        "--coinbase-recipient",
+        "0x1111111111111111111111111111111111111111111111111111111111111111",
+        "--min-da-gas-price",
+        "0",
+        "--da-gas-price-p-component",
+        "0",
+        "--da-gas-price-d-component",
+        "0",
+        "--starting-gas-price",
+        "1000",
+        "--gas-price-change-percent",
+        "10",
+        "--max-da-gas-price-change-percent",
+        "0",
```

Comment:
Why do we have these if it's supposed to be using `default_parameters`?

Reply:
resolved